app/templates/partials/gtm.html

```diff
@@ -1,12 +1,12 @@
 <!-- Google Tag Manager -->
-<script nonce="{{ request.csp_nonce }}">
```

Comment:
No problem with the gtm_auth changes to set-up the GTM environments with the boiler plate header and body code. Don't believe this change to the script nonce attribute should be included in this PR though. This should be considered in conjunction with fixing the problem with the Content Security Policy header. We shouldn't really be allowing inline scripts, if we do we should all know why.